HospitalFrontend/src/views/ObservePatientExaminations.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/toastify-js/src/toastify.min.css">
+    <div>
+        <h1>Your appointments</h1>
+        <div class="container w-50" v-if="loaded">
+            <div name="passed" class="container bg-light border border-gray">
+                <div class="text-left text-dark lead">Past appointments</div>
+                <ExaminationItem v-for="(examination, index) in getPastAppointments()"
+                                 v-bind:key="examination.id" v-bind:examination="examination"
+                                 v-bind:surveyCompleted="surveyStatuses[index]"></ExaminationItem>
+            </div>
+            <div class="container bg-light border border-gray" name="today">
+                <div class="text-left text-success lead">Today's appointments</div>
+                <ExaminationItem v-for="(examination, index) in getTodaysAppointments()"
+                                 v-bind:key="examination.id" v-bind:examination="examination"
+                                 v-bind:surveyCompleted="surveyStatuses[index]"></ExaminationItem>
+            </div>
+            <div class="container bg-light border border-gray" name="upcoming">
+                <div class="text-left text-info lead">Future appointments</div>
+                <ExaminationItem v-for="(examination, index) in getFutureAppointments()"
+                                 v-on:update-examinations="getExaminations()" v-bind:key="examination.id"
+                                 v-bind:examination="examination"
+                                 v-bind:surveyCompleted="surveyStatuses[index]"
+                ></ExaminationItem>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import axios from 'axios'
+import api from '../constant/api.js'
+import ExaminationItem from '../components/ExaminationItem.vue'
+import Toastify from 'toastify-js'
+
+export default {
+    name: "ObservePatientExaminations",
+    data: function () {
+        return {
+            examinations: [],
+            surveyStatuses: [],
+            loaded: false
+        }
+    },
+    components: {
+        ExaminationItem
+    },
+    mounted: function () {
+        this.getExaminations()
+    },
+    methods: {
+        getExaminations() {
+            axios.get(api.patientExaminations + '/patient/1')
+            .then(response => {
+                this.examinations = response.data
+                this.getSurveyStatuses()
+            })
+        },
+        getSurveyStatuses() {
+            let payload = []
+            this.examinations.forEach(examination => {
+                if (new Date(examination.timeInterval.start) < Date.now()) {
+                  payload.push(examination.id)
+                }
+            })
+            axios.post(api.patientExaminationsSurveyResponsesUrl, payload)
+            .catch(error => {
+                alert(error.response.data)
```

Comment:
Do you call alert intentionally or it should be toastify call?

Reply:
This shouldn't be here at all. Fixed.